source/jsonnet/england-wales/ccs/blocks/individual/past_usual_household_address.jsonnet

```diff
@@ -33,9 +33,9 @@ local question(title) = {
   ],
 };
 
-local nonProxyTitle = 'One year ago, on 13 October 2018, what was your usual address?';
+local nonProxyTitle = 'One year ago, on 21 March 2020, what was your usual address?';
 local proxyTitle = {
-  text: 'One year ago, on 13 October 2018, what was {person_name_possessive} usual address?',
+  text: 'One year ago, on 21 March 2020, what was {person_name_possessive} usual address?',
```

Comment:
These dates should be calculated. If it's not much work ideally as part of this ticket. If it's a fair bit more involved, it's worth creating another ticket to update them.

Reply:
I think it means adding new transform to runner and transform rules-definitions to validator, then updating eq-schemas. I would put it on a separate card.

Reply:
We could consider making it a placeholder for now and hard coding the date in the placeholder. That way we will have removed hardcoded dates from all strings.